spyder/plugins/projects.py

```diff
@@ -153,6 +154,9 @@ def register_plugin(self):
         self.sig_project_closed.connect(
             lambda v: self.editor.setup_open_files())
         self.recent_project_menu.aboutToShow.connect(self.setup_menu_actions)
+        self.run.connect(lambda fname: self.main.open_external_console(

```

Comment:
Does ``create_client_for_file`` executes the file? It seems that it opens a new console without executing the file